### DISCUSSION TITLE
3D enclosure to print?

### DISCUSSION THREAD

**Discussion by exocyt0sis** [PricelessToolkit/MailBoxGuard#25] [Q&A] [answered]
This looks like a real fun and useful project! I have a couple of questions, if you don't mind?

1. I notice that there's a 3D printed enclosure on the main Github page. Is the .stl-file available to download if you want to print it yourself?
2. What can you expect from the device in terms of partical range, if the radio signal from the gateway to the mailbox has to pass a few walls?

**Comment by PricelessToolkit** [comment, marked as answer]
Hi, 
1. STL files are in the 3D folder.
2. If your Gateway "Rx" antenna is not completely cr... it will work 99.9%.
